packages/plugin-lightningcss/src/index.ts

```diff
@@ -0,0 +1,7 @@
+export { pluginLightningcss } from './plugin';
+export { default as lightningcssLoader } from './loader';
+export { LightningCssMinifyPlugin } from './minimizer';
+export type * from './types';
```

Comment:
Wildcard exports is not recommended to use, please only export some public types.